app/javascript/components/auth-key-pair-cloud/auth-key-pair-cloud-form.schema.js

```diff
@@ -0,0 +1,48 @@
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+import { API } from '../../http_api';
+
+const providerOptions = () => API.get(`/auth_key_pair_cloud/ems_form_choices`).then(data => [
+  ...data.ems_choices.map(({ id, name }) => ({ value: id, label: name })),
+
+]);
+
+function createSchema() {
+  const fields = [
+    {
+      id: 'name',
+      name: 'name',
+      label: __('Name'),
+      component: componentTypes.TEXT_FIELD,
```

Comment:
For consistency, please put the `component` as the first attribute and keep the other attributes in the same order for each item.